cfgov/core/services.py

```diff
@@ -68,8 +72,18 @@ def generate_pdf(self):
                 self.get_filename())
             return response
 
-    def get(self, *args, **kwargs):
-        return self.generate_pdf()
+    def post(self, request):
+        form = CalenderPDFFilterForm(request.POST)
```

Comment:
@kurtw so i could wrap the request in a form, it makes getting individual values simpler & validation cleaner